ffc/backends/ufc/dofmap.py

```diff
@@ -122,6 +122,111 @@ def tabulate_facet_dofs(L, ir):
         return L.NoOp()
 
 
+def tabulate_dof_permutations(L, ir):
```

Comment:
Could this function more clearly separate (1) the concepts and (2) the code formatting (the `L` parts).